src/auth/project-config.ts

```diff
@@ -57,6 +62,13 @@ export class ProjectConfig {
    * This is based on the calling code of the destination phone number.
    */
   public readonly smsRegionConfig?: SmsRegionConfig;
+  private readonly multiFactorConfig_?: MultiFactorConfig;
```

Comment:
Don't you need this field in ProjectConfigClientRequest and ProjectConfigServerResponse?